doc/mf6io/mf6ivar/mf6ivar.py

```diff
@@ -145,7 +145,7 @@
 
 
 def parse_mf6var_file(fname):
-    f = open(fname, 'r')
+    f = open(fname, "r")
```

Comment:
presumably all these changes to mf6ivar.py are just are from black

Reply:
@langevin-usgs, I think they must be as I don't typically muck around in the already-existing python files other than to add lines needed to support new files (packages).  I definitely would not have made all those changes by hand.  Generally, if I touch a .py file, I'll run black on it because the ci will invariably catch something that I should've preemptively cleaned with black.  As I was on black version 23.1.0, I just updated to 23.12.0 and reran black and committed changes reflected in f76303a.  As none of the quote types changed, I'm thinking these must be the latest-greatest enforced by [black](https://github.com/psf/black).